File: coding_meetup_#7.js
function findSenior(list) {
  // thank you for checking out the Coding Meetup kata :)
  let oldestAge=list[0].age
  list.forEach(dev=>{
    if(dev.age>oldestAge) oldestAge=dev.age
  })
  console.log(list.filter(dev=>dev.age===oldestAge))
  return list.filter(dev=>dev.age===oldestAge)
  
}

File: coding_meetup_#7.py
def find_senior(lst):
#     print(lst)
    old_devs=[]
    oldest_dev=max(lst,key=lambda dev: dev['age'])
    print(oldest_dev)
    for dev in lst:
        if dev['age']==oldest_dev['age']:
            old_devs.append(dev)
    print(old_devs)
    return old_devs